types/wlr_foreign_toplevel_management_v1.c

```diff
@@ -405,6 +405,28 @@ void wlr_foreign_toplevel_handle_v1_set_fullscreen(
 	toplevel_send_state(toplevel);
 }
 
+static void send_parent(struct wlr_foreign_toplevel_handle_v1 *parent,
+		struct wl_resource *toplevel_resource) {
+	struct wl_client *client = wl_resource_get_client(toplevel_resource);
+	struct wl_resource *parent_resource = NULL;
+	if(parent) parent_resource = wl_resource_find_for_client(&parent->resources, client);
+	/* note: parent_resource will be NULL if this client does not hold
+	 * a handle for the parent or if a NULL parent was supplied */
+	zwlr_foreign_toplevel_handle_v1_send_parent(toplevel_resource,
```

Comment:
Do we really want null? https://github.com/swaywm/wlr-protocols/pull/52#issuecomment-705080313

Reply:
Not necessarily; at least I don't have a preference for either, I'll update this to only send NULL if the caller explicitly supplies a NULL parent.